README.md

```diff
@@ -58,5 +58,32 @@ export CUBIT_ROOT=path_to_cubit_root_directory
 To check if everything worked as expected, run the tests from within the `tests` directory
 ```bash
 cd path_to_cubitpy/tests
-pytest -q testing.py 
+pytest -q testing.py
 ```
+
+If you intend to develope CubitPy, please make sure to install the `pre-commit` hook within the python environment to follow our style guides:
+```bash
+pre-commit install
+```
+
+## Contributing
+
+Thank you for your interest in contributing! To keep things simple, please follow these steps:
+
+1. **Fork the Repository:** All contributions should be made from your own fork. Click "Fork" at the top of this repository, clone your fork to your local machine, and create a new branch for your work:
+   ```bash
+   git clone https://github.com/imcs-compsim/cubitpy.git
+   git checkout -b feature-or-bugfix-name
+   ```
+1. **Code Standards:** Before you are ready to contribute to MeshPy, please make sure to install the `pre-commit` hook within the python environment to follow our style guides:
```

Comment:
```suggestion
1. **Code Standards:** Before you are ready to contribute to CubitPy, please make sure to install the `pre-commit` hook within the python environment to follow our style guides:
```

Reply:
I guess CubitPy looked up too much to it's bigger brother 😂